embc-app/ClientApp/src/app/volunteer/pages/referral-view/referral-view.component.ts

```diff
@@ -58,7 +58,9 @@ export class ReferralViewComponent implements OnInit, OnDestroy {
           console.log('ERROR - invalid referral object = ', x);
           this.goHome();
         } else {
-          this.referral = x.referral;
+          // HACK for BE returning id instead of referralId
```

Comment:
This was changed on the FE request, do you prefer the BE will be changed back to referralId?

Reply:
That's a valid option too. Just LMK what you do so I can keep up :)

Reply:
your call (BFF)

Reply:
At the time of the request it was to work around a bug in a component built on the original data that I didn't have time to fix. I think that bug is fixed? If we pick a consistent way then I will be fine with whatever.
I'll wait for Severin to weight in.

Reply:
Let's not make further changes now. We can change this later.

Reply:
> your call (BFF)

BFF?

Reply:
Backend For Frontend

Reply:
> Backend For Frontend

I think you just made that up! :)